regression/tests/helpers.py

```diff
@@ -9,3 +11,37 @@ def visit_all(pages):
     for page in pages:
         print "Visiting: {}".format(page)
         page.visit()
+
+
+class LoginHelper():
+    """
+    Helper class to login to the studio.
+    """
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def get_login_email():
+        """
+        Return login email set in environment variable
+        """
+        login_email = os.environ.get('USER_LOGIN_EMAIL', 'not_set@nothing.com')
```

Comment:
I also feel like this logic should be generalized. (Perhaps I shoudln't be commenting on the tests...just paver?)